Assets/AnkrSDK/Examples/Scripts/LoggerEventHandler.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using AnkrSDK.Core.Events.Infrastructure;
+using Nethereum.RPC.Eth.DTOs;
+using UnityEngine;
+
+namespace AnkrSDK.Examples
+{
+	public class LoggerEventHandler : ITransactionEventHandler
```

Comment:
In my opinion it's not convinient approach by next reasons:
1) If I need only one event I still need to implement all of this events.
2) Better way to work with data that received to this methods, handle it exactly the same class that call contract method.